scss/widgets/material/slider/_index.scss

```
@@ -88,6 +88,7 @@ $material-slider-disabled-tooltip-top-margin: 5px;
       }
 
       .dx-popup-content {
+        overflow: hidden;
```

How can we test this? #
```suggestion
        overflow: hidden;
```